packages/xchain-utxo-providers/src/providers/blockbook/blockbook-api.ts

```
@@ -0,0 +1,124 @@
+  BroadcastDTO,
+  FeeEstimateDTO,
+  GetAddressInfo,
+  Transaction,
+  TxHashParams,
+} from './blockbook-api-types'
+
+const makeHeaders = (apiKey?: string) => (apiKey ? { 'api-key': apiKey } : {})
+
+/** Path segment encoding: `:` is valid unencoded in paths (RFC 3986). Avoid `%3A` in URLs/logs while keeping other escapes. */
+const encodeAddressPathSegment = (address: string) =>
+  encodeURIComponent(address).replace(/%3A/g, ':')
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)